src/esp/assets/GenericInstanceMeshData.cpp

```diff
@@ -40,10 +42,32 @@ namespace assets {
 namespace {
 template <typename T>
 void copyTo(std::shared_ptr<tinyply::PlyData> data, std::vector<T>& dst) {
-  dst.resize(data->count);
+  dst.resize(data->buffer.size_bytes() / sizeof(T));
```

Comment:
This was done like that for a reason -- it makes sure that you didn't try to read a `uint32_t` field as `uint16_t`, which would be bad.

Reply:
`ASSERT(data->count*sizeof(T) == data->buffer.size_bytes());` then?

Reply:
Yeah, that'll do it too :)

Reply:
Yeah, this was not ideal but was required since cpu_ibo_ is now a std::vector<uint32_t> instead of a std::vector<vec3ui>. This was needed in order to use cpu_ibo_ with MeshTools::combineIndexedArrays.

Will use the new ASSERT suggested by mosra.